app/views/spree/checkout/existing_payment/_gateway.html.erb

```diff
@@ -1,9 +1,9 @@
-<tr id="<%= dom_id(wallet_payment_source, 'spree')%>">
+<tr id="<%= dom_id(wallet_payment_source, 'spree')%>" class="<%= cycle('even', 'odd') %>">
```

Comment:
Do we really need to add this class? If it's for style purposes, I think we can do this with css.

Reply:
Removed, those classes were unused. 👍